examples/rpt/count_calculation/rpt_count_calculation.prm

```diff
@@ -0,0 +1,37 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# RPT Monte Carlo technique
+#---------------------------------------------------
+subsection rpt parameters
+    set particle positions file          = positions1.particle
+    set export counts                    = true
+    set iteration number                 = 1000000
+    set random number seed               = auto
+    set reactor radius       			 = 0.1
+    set peak-to-total ratio  			 = 0.4
+    set sampling time        			 = 1
+end
+	
+# --------------------------------------------------
+# Initial value of parameters for tuning
+#---------------------------------------------------
+subsection parameter tuning
+	set tuning                           = false	
+    set dead time       				 = 1e-5
+    set activity  						 = 2e6
+    set gamma-rays emitted        		 = 2
+    set attenuation coefficient reactor  = 10
```

Comment:
This attenuation coefficient is  related to reactor (not only reactor body) and material inside, I think reactor content is a better word but not the best representative maybe.

Reply:
Let's lump all "RPT ish" parameters together for the time being. This will make life simpler for now.

Reply:
I did explain a bit more as comments in the code itself, but good point, it will be important to explain it also in documentation.